test/e2e/e2e.go

```diff
@@ -639,22 +634,9 @@ func checkConditionStatus(instance *metallbv1beta1.MetalLB) string {
 	return ""
 }
 
-func decodeYAML(r io.Reader, obj interface{}) error {
-	decoder := yaml.NewYAMLToJSONDecoder(r)
-	return decoder.Decode(obj)
+func getMetalLB() *metallbv1beta1.MetalLB {
```

Comment:
Why not loading the file anymore?

Reply:
The path to the file is invalid when importing the tests in other tests suites (for example CNF tests).

Reply:
Can you leave it as it is and eventually provide a handle to override via env variable?
There's value in validating the local files we don't want to loose

Reply:
There are some pitfalls in this. Go vendor only gets dirs with go files, and even once inside the vendor dir it would not be available in the build binary, so we don't get this by default.
Should we get this manually into our image in the dockerfile?

Reply:
As discusssed offline, modified to use an env var to bypass the loading from file.